test/index.js

```diff
@@ -39,10 +41,34 @@ test('Simple aliasing', (t) => {
   t.is(resolved3, 'global');
 });
 
+test('RegExp aliasing', (t) => {
+  const result = alias({
+    entries: [
+      {find:new RegExp('fo.*'), replacement:'foobar2019', isRegEx:true},
+      {find:new RegExp('.*pony.*'), replacement:'i/am/a/barbie/girl', isRegEx:true},
+      {find:new RegExp('^test/$'), replacement:'this/is/strict', isRegEx:true}
```

Comment:
Would be nice to also use RegExp literals as in the Readme?

Reply:
agreed.